### ISSUE TITLE
Single Doc for multiple APIs

### ISSUE THREAD

**Issue by Wataker0912** [RicoSuter/NSwag#2717]
Hello, search didn't yield much results, I was curious if it would be possible to maintain a centralized swagger doc with multiple APIs/ controllers across separate C# solutions?  Thanks!

**Comment by Wataker0912** [comment]
ok cool, I was able to add a master proejct with references to each controller, although they seem to load in just one long scrolling page, no real organization.  Is there a way I can organize them by name or title or something? Thanks!

**Comment by RicoSuter** [comment]
Arent they grouped by tag (tag == controller name)?

**Comment by Wataker0912** [comment]
Yes they are, sorry I meant organize them as in a 'Title' for each set of controllers which come from the same project.  So if I had 5 Controllers from one project called 'Pets', list them all under 'Pets, 2 from 'Fish', put them under 'Fish' etc.  Thanks!

**Comment by RicoSuter** [comment]
You can write an operation processor and change the operation tags there: 
https://github.com/RicoSuter/NSwag/wiki/Document-Processors-and-Operation-Processors
